app/components/global/layout/DropdownNavLayout.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+import NavLink from '../../ui/atoms/NavLink'
+import Select, { createFilter, components } from 'react-select'
+
+const Root = styled.div``
+
+const DropdownNavLink = styled(NavLink)`
+  display: block;
+  padding: ${th('space.1')};
+`
+
+class DropdownNavLayout extends React.Component {
```

Comment:
If you find it easier to create a Nav-specific dropdown component first (issue #820), that's fair enough.

But just be aware that we'll need to refactor this later and pull out the code which is generic to all dropdowns, regardless of the content in the list. Then use that generic dropdown (issue #821) in order to build the version in the dashboard (issue #786). So you might be better off just making the generic one first? Just a suggestion.

Either way, it might be useful to have this component in the `ui/atoms` folder rather than `global/layout`?